workloads/librispeech/prepare_data.py

```diff
@@ -111,7 +111,7 @@ def extract_spect_mvn(audio_path):
 if __name__ == '__main__':
   data_dir = sys.argv[1]
 
-  trans_dir = os.getcwd() + 'data'
+  trans_dir = os.getcwd() + '/data'
```

Comment:
for both of these, can we instead use `os.path.join` (imported via `import os.path`)?

Reply:
done